meerk40t/core/elements.py

```diff
@@ -8943,6 +8949,15 @@ def classify(self, elements, operations=None, add_op_function=None):
                 because of a lack of classification options.
         @return:
         """
+        def emptydebug(value):
+            return
+
+        # I am tired of changing the code all the time, so let's do it properly
```

Comment:
Do it *really* properly. `debug = context.channel("classification")`

Then it's a channel. You can print that, save to disk, open in the console to be watched.